.github/ISSUE_TEMPLATE/fix-content.yml

```diff
@@ -0,0 +1,34 @@
+name: Fix Content
+description: Report typos or minor documentation issues
+title: '[FIX]: '
+labels: ['update']
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to submit this issue. Your input is appreciated.
+  - type: textarea
+    id: issue-location
+    attributes:
+      label: Where is the issue?
+      description: Link to the documentation page(s) where you identified this issue.
+      placeholder: 
+  - type: textarea
+    id: issue-text
+    attributes:
+      label: What's in the docs?
+      description: Provide the text or content currently in the docs.
+      placeholder:
+    validations:
+      required: true
+  - type: textarea
+    id: text-resolution
+    attributes:
+      label: What should be in the docs?
```

Comment:
Not everyone raising an issue will know what the fix is. Maybe we can have a more general question of "what is the issue?" with options like "missing content," "outdated content," and "inaccurate information." And provide a space to describe the issue in more detail.